requirements.txt

```diff
@@ -28,7 +28,6 @@ pydicom==2.4.4
 pydicom-seg==0.4.1
 pynetdicom==2.0.2
 pynrrd==1.0.0
-opencv-python-headless==4.9.0.80
```

Comment:
Hi @tangy5 , @SachidanandAlle , do we still need to update some tutorials or docs to let users install the OpenCV package as it's optional package now.

Thanks.